src/gist-inject.less

```diff
@@ -9,8 +9,12 @@
         max-width: @breakpoint - @width !important
     }
 
+    .container-xl {
+        max-width: none !important;
+    }
+
     .container-lg {
-        .max-width(120px);
+        max-width: none !important;
```

Comment:
Can we keep this at 120px? Right now it's just a tad _too_ wide 😅

Reply:
That's great.